atum/src/main/scala/za/co/absa/atum/AtumImplicits.scala

```diff
@@ -270,9 +270,12 @@ trait AtumImplicitsBase {
     }
 
     /**
-     * The method sets an additional info in metadata of the _INFO file with key - value pair
+     * Sets an additional info in metadata of the _INFO file
+     * @param keyAndValue - the pair of the _key_ and _value_ to add to the _INFO file, in a form of tuple
+     * @param replaceIfExists - flag to indicate if the value should be replaced in the case the key already exists
+     * @return - the original `DataFrame` the method was called upon
      */
-    def setAdditionalInfo(keyAndValue: (String, String), replaceIfExists: Boolean = false): DataFrame = {
+    def setAdditionalInfo(keyAndValue: (String, String), replaceIfExists: Boolean = false): Dataset[Row] = {
```

Comment:
FYI: `DataFrame` is a synonym for 'Dataset[Row]` (dataset of type row]. If you Ctrl+Click `DataFrame` you will see it's defined as `type DataFrame = Dataset[Row]`. 🤓